source/geometry/Quaternion.ooc

```diff
@@ -346,6 +332,4 @@ Quaternion: cover {
 	}
 	kean_math_quaternion_new: unmangled static func (w, x, y, z: Float) -> This { This new(w, x, y, z) }
 }
-operator * (value: Float, other: Quaternion) -> Quaternion {
-	other * value
-}
+operator * (value: Float, other: Quaternion) -> Quaternion { other * value }
```

Comment:
I considered that, but since it's nothing below the operators, I let them be. Should I insert a new line?